src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -25,17 +25,17 @@ final class DateTimeNormalizer implements NormalizerInterface, DenormalizerInter
     public const FORMAT_KEY = 'datetime_format';
     public const TIMEZONE_KEY = 'datetime_timezone';
     public const CAST_KEY = 'datetime_cast';
+    public const SUPPORTED_TYPES_KEY = 'supported_types';
```

Comment:
```suggestion
    public const SUPPORTED_TYPES_KEY = 'datetime_supported_types';
```

to avoid conflicts with other normalizers